lib/types.js

```diff
@@ -41,10 +41,65 @@ const exclude = function (types, excludes) {
   return typesCopy;
 }
 
+const excludeImmutable = function (types, excludes) {
+  if (!excludes || excludes.length < 1) {
+    return types;
+  }
+
+  let mutations = [];
```

Comment:
Sorry this variable name is confusing me. Is this strictly mutations? Doesn't appear so.

Reply:
hastily named 😄 
it's more of a "to do" list of mutations to make, so maybe `mutationsToMake`?